src/components/PupilComponents/QuizAttribution/QuizAttribution.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { OakThemeProvider, oakDefaultTheme } from "@oaknational/oak-components";
+import { render } from "@testing-library/react";
+
+import { isImage } from "../QuizUtils/stemUtils";
+
+import { QuizAttribution } from "./QuizAttribution";
+
+import {
+  mcqImageAnswers,
+  mcqTextAnswers,
+  questionStem,
+  questionStemWithImage,
+  quizQuestions,
+} from "@/node-lib/curriculum-api-2023/fixtures/quizElements.fixture";
+
+describe(QuizAttribution, () => {
+  it("displays attribution for any image in the question stem", () => {
+    const { queryByText } = render(
+      <OakThemeProvider theme={oakDefaultTheme}>
+        <QuizAttribution
+          questionData={{
+            ...quizQuestions[0]!,
+            answers: {
+              "multiple-choice": mcqTextAnswers,
+            },
+            questionStem: questionStemWithImage,
+          }}
+        />
+      </OakThemeProvider>,
+    );
+
+    expect(queryByText("test attribution picture")).toBeInTheDocument();
+  });
+
+  it("displays attribution for images in answers", () => {
+    const { getByText, queryByText } = render(
+      <OakThemeProvider theme={oakDefaultTheme}>
+        <QuizAttribution
+          questionData={{
+            ...quizQuestions[0]!,
+            answers: {
+              "multiple-choice": mcqImageAnswers.map((choice, i) => ({
+                ...choice,
+                answer: choice.answer.filter(isImage).map((answer) => ({
+                  ...answer,
+                  image_object: {
+                    ...answer.image_object,
```

Comment:
Not a fan of this nesting but it's better than copying the whole fixture across. Come utilities for generating complex fixtures wouldn't go amiss in the future.